examples/plotWidget.py

```diff
@@ -41,7 +41,7 @@
 
 from silx.gui import qt
 
-from silx.gui.plot import PlotWidget
```

Comment:
Why switching to `PlotWindow` while the name of the example is `plotWidget`?

Reply:
My bad, this is a leftovers from some test :(